e2e/nomostest/wait_for_sync.go

```diff
@@ -186,7 +187,11 @@ func (nt *NT) WatchForSync(
 	}
 
 	predicates := []testpredicates.Predicate{
+		// Status reflects latest spec changes
 		testpredicates.HasObservedLatestGeneration(nt.Scheme),
+		// Not Terminating, Reconciling, or Stalled
```

Comment:
Can you elaborate more on this comment to make it more useful?

Reply:
What are you looking for? Those are the predicates it's waiting for as part of the status evaluation.

Reply:
Can you make this comment into a sentence?

Reply:
done